rmw_opensplice_cpp/src/rmw_wait.cpp

```diff
@@ -196,6 +272,23 @@ rmw_wait(
     }
   }
 
+  std::unordered_set<DDS::StatusCondition *> status_conditions;
+  // gather all status conditions with set masks
+  rmw_ret_t ret_code = __gather_event_conditions(events, status_conditions);
+  if (ret_code != RMW_RET_OK) {
+    return ret_code;
+  }
+  if (!status_conditions.empty()) {
+    // enable a status condition for each event
+    for (auto * status_condition : status_conditions) {
```

Comment:
I mentioned this elsewhere (different pr), but the `*` isn't strictly needed. If you want to leave it for clarity, that's fine, but I just wanted to point it out.

Reply:
I have removed the `*` to be consistent with `rmw_connext_shared_cpp`.